### ISSUE TITLE
Add REUSE support in the GitHub licensee library

### ISSUE THREAD

**Issue by ottok** [fsfe/reuse-tool#901]
Currently it seems that the [GitHub licensee Ruby library](https://github.com/licensee/licensee/tree/main/lib/licensee/matchers) they use for detecting licenses does not read the path `LICENSES/`. People hosting their code on GitHub probably value having GitHub detect their license correctly. Thus it might help drive adoption if the licensee matchers at https://github.com/licensee/licensee/tree/main/lib/licensee/matchers were extended to read the places REUSE recommends developers to use.

I am posting this for visibility while I am fully aware that this is not a bug in the reuse-tool itself.

**Comment by Mikaela** [comment]
https://github.com/licensee/licensee/issues/635 looks like a related issue on their side although it's only about SPDX identifiers rather than the LICENSES/ directory.

**Comment by mxmehl** [comment]
There have been a number of upstream issues about this already, and also discussions via Twitter IIRC. Mike Linksvayer has made it quite clear that they only implement this when a pull request is made by a third party.

The REUSE team doesn't currently plan to do that but we welcome any initiative!

**Comment by sdavids** [comment]
> Mike Linksvayer has made it quite clear that they only implement this when a pull request is made by a third party.

The PR should target which repo?

**Comment by nicorikken** [comment]
Indeed, unless GitHub has changes libraries in the mean time, but I wouldn't think so. Regardless, it would be good to have it in this commonly used library.

**Comment by ottok** [comment]
I see https://github.com/licensee/licensee/issues/490 was marked stale and closed as "completed" even though it wasn't resolved. This issue isn't resolved either, so logically it should stay open until resolved. One would assume that closing issues that aren't actually fixed just leads to them being filed again as new issues?
